GUI/QtQuick/qml/main.qml

```diff
@@ -198,13 +198,16 @@ ApplicationWindow {
                 font.pointSize: 16
                 Layout.fillHeight: true
                 onClicked: {
-                    bsApp.requestFeeSuggestions()
-                    topMenuBtnClicked(btnSend)
-                    //stack.push(sendPage)
-                    send_popup.init()
-                    send_popup.show()
-                    send_popup.raise()
-                    send_popup.requestActivate()
+                    if (walletBalances.rowCount > 0)
```

Comment:
Corresponding buttons should be disabled, too, I guess?

Reply:
Dont sure, maybe we will add message box if no wallets - "Please add at least 1 wallet to can sending bitcoins"